snippets/javascript/framer/framer.json

```diff
@@ -0,0 +1,176 @@
+{
+  "framer-import": {
+    "prefix": ["framer import", "framer"],
+    "body": ["import { motion } from 'framer-motion'", ""],
+    "scope": "javascript,javascriptreact",
+    "description": "Framer provides helpers for advanced physics-based animation, complex touch-based gestures and common components for scrolling, paging and interface flows. It’s designed to allow beginners to explore digital product ideas without boundaries."
+  },
+  "framer-motion": {
+    "prefix": ["framer motion", "motion"],
+    "body": [
+      " <motion.${1:div}",
+      "    initial=\"hidden\"",
+      "    animate=\"visible\"",
+      "    variants={${2:variants}}",
+      "  ></motion.${1:div}>"
+    ],
+    "scope": "javascript,javascriptreact",
+    "description": "Motion shares the simple animation and gesture API as the Framer X library, while maintaining HTML and SVG semantics. This makes it simple to port your high-fidelity prototypes to a production-ready environment, or make something completely new!"
```

Comment:
Same here, the description should help you get understanding about the snippet you will use. Better to leave it  blank than use just some description from the website.